driver/src/test/resources/discover_failed.script

```diff
@@ -3,7 +3,7 @@
 !: AUTO HELLO
 !: AUTO GOODBYE
 
-C: RUN "CALL dbms.cluster.routing.getRoutingTable($context)" {"context": {}} {}
+C: RUN "CALL dbms.cluster.routing.getRoutingTable({context})" {"context": {}} {}
```

Comment:
Isn't the `{context}` parameter naming syntax [deprecated](https://neo4j.com/docs/cypher-manual/current/syntax/parameters/#cypher-parameters-introduction) in favor of `$context`? 

Valid for rest of the stub scripts as well.